xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -41,10 +41,7 @@ struct MemoViewerDetailView: View {
                     notify: notify
                 )
             case .notFound:
-                MemoViewerDetailNotFoundView(
-                    backlinks: store.state.backlinks,
```

Comment:
I think we want to include backlinks with this view (no dead ends). That was the original intent of passing them in, but I don't know if it was actually ever implemented.

Non-blocking, but we should file a bug to track.

Reply:
Wasn't sure on this one but I've restored the backlinks view. 

<img width="386" alt="image" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/cbde3281-31e6-4ed4-920b-9b4b69bab3fe">

Since the only way to reach the memo viewer empty state is via 3P content we don't see any backlinks yet, pending #323.